src/js/heuristicblocking.js

```diff
@@ -187,9 +187,9 @@ HeuristicBlocker.prototype = {
     // not-yet-blocked domain for DNT policy.
     // We check heuristically-blocked domains in webrequest.js.
     if (!skip_dnt_check) {
-      window.setTimeout(function () {
+      setTimeout(function () {
         badger.checkForDNTPolicy(tracker_fqdn);
-      }, 10);
+      }, 0);
```

Comment:
Sorry if I'm misunderstanding here -- why setTimeout if there's no timeout? Is this just to make the call async?

Reply:
Yep: https://johnresig.com/blog/how-javascript-timers-work/

Reply:
Sorry if I'm misunderstanding here -- why setTimeout if there's no timeout? Is this just to make the call async?

Reply:
Thanks! Mind adding a short comment to that effect?